modules/aws/rds/variables.tf

```diff
@@ -54,17 +70,19 @@ variable "identifier" {
 
 variable "visibility" {
   description = "Visibility of the rds instance"
+  default = "private"
   type        = string
 }
 
 variable "vpc" {
   description = "All vpc info"
   type = object({
     name = string
-    id   = string
+    vpc_id   = string
     public_subnets = list(string)
     private_subnets = list(string)
-    database_subnets_cidr_blocks = list(string)
+    database_subnets = list(string)
```

Comment:
@suryaoruganti We are not using below variables in `main.tf` file of rds module.
- database_subnets
- default_security_group_id

Reply:
Ack, but not an issue